src/Workleap.DomainEventPropagation.Tests/Publishing/EventPropagationPublisherOptionsValidatorTests.cs

```diff
@@ -11,51 +11,29 @@ namespace Workleap.DomainEventPropagation.Tests.Publishing;
 public class EventPropagationPublisherOptionsValidatorTests
 {
     [Theory]
-    [InlineData(null, null, null)]
-    [InlineData(" ", "whatever", "http://workleap.com")]
-    [InlineData("unrecognizedTopic", "whatever", "http://workleap.com")]
-    [InlineData("Signup", null, "http://workleap.com")]
-    [InlineData("InvalidTopic", "AccessKey", "http://workleap.com")]
-    [InlineData(" ", "AccessKey", "http://workleap.com")]
-    [InlineData(null, "AccessKey", "http://workleap.com")]
-    [InlineData("Organization", null, "http://workleap.com")]
-    public void GivenInvalidEventPropagationConfigurations_WhenValidatingOption_ThenFail(string topicName, string topicAccessKey, string topicEndpoint, bool useTokenCrential = false)
+    [InlineData("topicName", " ", "http://topicurl.com", true, true)]
```

Comment:
Similar here, since we only currently validate the accessKey and TokenCredentials. I kept the data to only the possible combinations.